Modules/Microsoft365DSC/DSCResources/MSFT_IntuneWifiConfigurationPolicyIOS/MSFT_IntuneWifiConfigurationPolicyIOS.schema.mof

```diff
@@ -18,14 +18,15 @@ class MSFT_IntuneWifiConfigurationPolicyIOS : OMI_BaseResource
     [Write, Description("Connect automatically")] Boolean ConnectAutomatically;
     [Write, Description("Connect when network name is hidden")] Boolean ConnectWhenNetworkNameIsHidden;
     [Write, Description("Disable the MAC address randomization.")] Boolean DisableMacAddressRandomization;
+    [Write, Description("If the pre shared key should be updated, even if the policy is already equal.")] Boolean ForcePreSharedKeyUpdate;
     [Write, Description("Network name")] String NetworkName;
     [Write, Description("Pre shared key")] String PreSharedKey;
     [Write, Description("Proxy automatic configuration url")] String ProxyAutomaticConfigurationUrl;
     [Write, Description("Proxy manual address")] String ProxyManualAddress;
     [Write, Description("Proxy manual port")] UInt32 ProxyManualPort;
     [Write, Description("Proxy settings"), ValueMap{"none","manual","automatic"}, Values{"none","manual","automatic"}] String ProxySettings;
     [Write, Description("SSID")] String Ssid;
-    [Write, Description("Wi-Fi security"), ValueMap{"open","wpaPersonal","wpaEnterprise","wep","wpa2Personal","wpa2Enterprise"}, Values{"open","wpaPersonal","wpaEnterprise","wep","wpa2Personal","wpa2Enterprise"}] String WiFiSecurityType;
+    [Write, Description("Wi-Fi security"), ValueMap{"open","wpaPersonal","wep"}, Values{"open","wpaPersonal","wep"}] String WiFiSecurityType;
```

Comment:
Removing values from the ValidateSet would be considered as breaking change. Any chance to keep the previous elements and wirte a warning about them being deprecated?